app/src/main/res/drawable/icons8_book_64.xml

```diff
@@ -0,0 +1,18 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Will need to create a new svg from Figma to correctly show the icon on splashscreen.